src/index.html

```diff
@@ -11,7 +11,167 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="post"
+      onsubmit="onSubmit()">
```

Comment:
Fix that on all code

```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      onsubmit="onSubmit()"
    >
```

Reply:
I hope, I did not miss any mistakes